config/initializers/_dmproadmap.rb

```diff
@@ -122,6 +122,10 @@ class Application < Rails::Application
     config.x.application.require_contributor_name = false
     config.x.application.require_contributor_email = false
 
+    # Defines if Guidances/Comments in toggleable & if it's opened by default
+    config.x.application.guidance_comments_toggleable = true
+    config.x.application.guidance_comments_opened = true
```

Comment:
Can you change the name of this variable to `guidance_comments_opened_by_default`?